functions/UpdateBookingSlot-BookToISV/run.csx

```diff
@@ -104,7 +103,7 @@ public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, CloudT
             // Compile and Write the mail record away to the ServiceBus if this is enabled for this TE (if the TE master record is populated);
 
             // Find the TE master data record 
-            technicalevangelist chkte = (from te in tblte.CreateQuery<technicalevangelist>() select te).Where(e => e.RowKey == bsout.TechnicalEvangelist && e.PartitionKey == "ALL").FirstOrDefault();
+            TechnicalResource chkte = (from te in tblte.CreateQuery<TechnicalResource>() select te).Where(e => e.RowKey == bsout.TechnicalEvangelist && e.PartitionKey == "ALL").FirstOrDefault();
```

Comment:
no problem, you’d also have to change the backing azure table name and migrate the existing records if you re-created it 😉